File: Beecrowd/2630 Escala de Cinza/main.c
#include <stdio.h>
#include <string.h>

int main(){
    int t, r, g, b, i = 1,m, p;
    char str[5];

    scanf("%d", &t);

    while(t--){
        scanf("%s", str);
        scanf("%d %d %d", &r, &g, &b);

        if(strcmp(str, "max") == 0){
            if(r > g && r > b)
                p = r;
            else if(g > r && g > b)
                p = g;
            else
                p = b;
        }
        else if(strcmp(str, "min") == 0){
            if(r < g && r < b)
                p = r;
            else if(g < r && g < b)
                p = g;
            else
                p = b;
        }
        else if(strcmp(str, "mean") == 0)
            p = (r + g + b) / 3;
        else
            p = (r * 0.3) + (g * 0.59) + (b * 0.11);

        printf("Caso #%d: %d\n", i++, p);
    }

    return 0;
}

File: Beecrowd/2630 Escala de Cinza/main.py
t = int(input())
i = 1

for _ in range(t):
    std = input()
    lista = [int(_) for _ in input().split()]

    if std == "max":
        p = max(lista)
    elif std == "min":
        p = min(lista)
    elif std == "mean":
        p = (lista[0] + lista[1] + lista[2]) // 3
    else:
        p = int((lista[0] * 0.3) + (lista[1] * 0.59) + (lista[2] * 0.11))
    
    print(f"Caso #{i}: {p}")

    i += 1